packages/data-models/deployment.model.ts

```diff
@@ -56,6 +56,7 @@ export interface IDeploymentConfig {
     /** filter function that receives basic file info such as relativePath and size. Default `(fileEntry)=>true`*/
     assets_filter_function: (fileEntry: IContentsEntry) => boolean;
   };
+  favicon_asset?: string;
```

Comment:
nit(non-blocking)
Should we consider nesting this within some sort of `web` config/property given that is the only context where this would apply?